src/js/dataMapping/covid19/awardSpendingAgency/awardSpendingAgencyTableTabs.js

```diff
@@ -0,0 +1,59 @@
+export const awardSpendingAgencyTableTabs = [
```

Comment:
We should be able to delete this array and use a more minimal version `awardTypeTabs` from `dataMapping/covid19/covid19`